README.md

```diff
@@ -13,3 +13,69 @@ For installation instructions see [INSTALL.md](https://github.com/hypothesis/tox
 
 For how to set up a tox-faster development environment see
 [HACKING.md](https://github.com/hypothesis/tox-faster/blob/main/HACKING.md).
+
+Speedups
```

Comment:
This is the contents of `.cookiecutter/includes/README.md` being appended to the standard `README.md` by the cookiecutter (`make template`).